code/framework/api/src/main/java/io/cattle/platform/api/auth/Identity.java

```diff
@@ -8,9 +9,12 @@
 import org.apache.commons.lang3.builder.ToStringBuilder;
 import org.springframework.util.DigestUtils;
```

Comment:
ah.... don't import spring.  don't need to change, but don't import spring classes in the future.